controllers/nc_news_controllers.js

```diff
@@ -0,0 +1,13 @@
+const {
+    selectTopics
+} = require('../models/nc_news_models.js')
+
+const getTopics = (req, res, next) => {
+    selectTopics().then((rows) => {
+        res.status(200).send({status:200, rows: rows})
```

Comment:
Nice to see an object being sent back, but we shouldnt be including the status code within this object and only need to send the data back. The status code is already being sent back because we are using res.status(...) to do this.